### DISCUSSION TITLE
渲染不同颜色标注

### DISCUSSION THREAD

**Discussion by OoOoYY** [zouyaoji/vue-cesium#244] [Q&A] [answered]
![1659602240553](https://user-images.githubusercontent.com/65013993/182803009-f03cfed6-881e-4c1f-bee6-cb8ab0a67fcf.png)
![1659602347587](https://user-images.githubusercontent.com/65013993/182803244-5af33afb-83e5-491b-814b-2253f72dbf31.png)
labelOpts可以支持数组吗，一个区域一个中心点，区域不同颜色，中心点不同名称

**Comment by OoOoYY** [comment, marked as answer]
区域点击事件

**Reply by zouyaoji** [reply]
可以对每一个点、线、面单独设置的。
通过模板引用来获取到绘制的数组
drawingsRef.getDrawingActionInstances()[3].cmpRef.value.renderDatas
